istio-cni/values_gke.yaml

```
@@ -1,6 +1,6 @@
 cni:
-  hub: docker.io/tiswanso
-  tag: v0.1-dev
+  hub: gcr.io/istio-release
```

```suggestion
  hub: gcr.io/istio-testing
```

We need to apply this to values.yaml as well I think

We can probably just delete values_gke.yaml, to avoid duplication. Its wrong to use it already since its not properly kept in sync